book/website/community-handbook/infrastructure/infrastructure-pathways.md

```diff
@@ -1,19 +1,10 @@
 (ch-infrastructure-pathways)=
 # Pathways
 
-The {ref}`ch-pathways` presents multiple curated sets of chapters to allow role and theme-based entry points to the book.
+The [](#ch-pathways) presents multiple curated sets of chapters to allow role and theme-based entry points to the book.
 This feature allows multiple entry points that people can select to find information most relevant to them.
```

Comment:
Can I move this to the landing page?

Reply:
I would prefer not to. There aren't concise tiles for pathways like we previously put on the welcome page. So, they are currently collected in a section.

Reply:
If the latter - there's an empty heading on the homepage which needs removing.

Reply:
Sorry - that was my mistake. Please feel free to remove the text from the other file.